TGX Files/Data/ObjectData/units/PIONEER.INI

```diff
@@ -1,5 +1,5 @@
 [ObjectData]
-ProperName                  =   STRING_4551_Pioneer
+ProperName                  =   STRING_6163_Journeyman
```

Comment:
Not sure about this change. Mechanically it's fine but we need to adjust descriptions and the name of the tech so we can retain the Pioneer name. Not only does calling the advanced settler a Journeyman not make as much sense, there's also the filename staying Pioneer that may make this unnecessarily confusing

Reply:
Thematically, I'd been thinking that the highly trained masons and craftsmen produced by the Journeymen's guild would be more proficient at building towns than your average settler, in addition to aiding in construction of infrastructure (components) in existing settlements. This especially fits with the unit requiring a masonry guild. 
I feel like the name Pioneer puts a lot of emphasis on rugged explorers, rather than efficient builders.
All that being said, I don't feel strongly about keeping the current version. Mostly wanting to explain the reasoning in case it wasn't clear